lib/Fetcher/Client/LegacyGuzzleClient.php

```diff
@@ -47,8 +47,9 @@ public function getResponse(string $url, \DateTime $modifiedSince) : ResponseInt
     {
         try {
             $options = [
+                $modifiedSince->setTimezone(new \DateTimeZone('GMT'));
```

Comment:
Move `$modifiedSince->setTimezone(new \DateTimeZone('GMT'));` before $options and probably before the try-catch-blocks.